player.py

```diff
@@ -6,7 +6,7 @@ def __init__(self,piece):
     def get_name(self):
         '''asks for the player's name and stores it'''
         self.name = input("What is your name?")
-        return self.name
+        return self.name # unnecessary
```

Comment:
You assigned a value to self.name on the previous line.  No need to return anything.